contributors/design-proposals/pod-priority-resourcequota.md

```diff
@@ -0,0 +1,77 @@
+# Priority in ResourceQuota
+
+@resouer @bsalamat @davidopp
+
+Aug 2017
+  * [Objective](#objective)
+    * [Non-Goals](#non-goals)
+  * [Background](#background)
+  * [Overview](#overview)
+  * [Detailed Design](#detailed-design)
+    * [Expected behavior of ResourceQuota apply to pod](#resourcequota-apply-to-pod)
+    * [Expected behavior of ResourceQuota admission controller](#resourcequota-admission-controller)
+
+
+## Objective
+
+This feature is designed to make ResourceQuota become priority aware, three sub-tasks are included.
+
+* Add a field to ResourceQuota that specifies a priority class name.
+* Incorporate priority in quota checks.
+* ResourceQuota admission controller to check ResourceQuota priority class and resolve the priority class to its integer value.
+
+### Non-Goals 
+
+* Add priority in Pod spec (this is implemented in: #45610)
+* Incorporate priority in pod scheduling logic.
```

Comment:
This is probably not needed here. It is obvious that this design doc is addressing a different topic.